src/MLJDecisionTreeInterface.jl

```diff
@@ -22,14 +22,20 @@ Base.show(stream::IO, c::TreePrinter) =
     print(stream, "TreePrinter object (call with display depth)")
 
 
+# add a new variable to struct a la
+const feature_importance_options = Dict(:impurity => DecisionTree.impurity_importance,
+                                        :split => DecisionTree.split_importance,
+                                     #   :permutation => DecisionTree.permutation_importance,
+                                    )
```

Comment:
Do we really need the Dictionary here? It might increase load time and we don't have any performance gain from it, since it would be of type `Dict{Symbol, Function}`. 
So I say using explicit if statements would perform better in this case.